pkg/controllers/gracefuleviction/evictiontask.go

```diff
@@ -108,6 +108,9 @@ func nextRetry(tasks []workv1alpha2.GracefulEvictionTask, timeout time.Duration,
 	retryInterval := timeout / 10
 
 	for i := range tasks {
+		if tasks[i].SuppressDeletion != nil {
```

Comment:
I guess we might need a comment to explain why we need to ignore this kind of task.

Reply:
Update!